dashboard/package.json

```diff
@@ -8,8 +8,8 @@
     "build": "NODE_ENV=production webpack --config src/config/webpack.app.js",
     "lint": "eslint \"src/**\"",
     "start": "hjs-dev-server src/config/webpack.app.js",
-    "test": "wdio test/conf/wdio.local.js",
-    "test-browserstack": "wdio test/conf/wdio.browserstack.js",
+    "test": "wdio test/conf/wdio.local.js --suite base",
+    "testExtended": "EXTENDED=1 wdio test/conf/wdio.local.js",
```

Comment:
This actually has two side effects. Running all tests (aka "extended") by not specifying a suite, and configuring the tests to [_not_ run in parallel for the extended suite](https://github.com/chain/chain/pull/478/files#diff-1019fea1af46f77a3038852a0e2a0665R51).

Since the extended tests involve empty states and core resets, parallelizing them with normal tests can result in random failures when a single core is being subjected to multiple tests.